src/hyperfine/format.rs

```diff
@@ -6,6 +6,34 @@ pub enum Unit {
     MilliSecond,
 }
 
+impl Unit {
+    /// The abbreviation of the Unit.
+    pub fn short_name(&self) -> String {
+        match *self {
+            Unit::Second => String::from("s"),
+            Unit::MilliSecond => String::from("ms"),
+        }
+    }
+
+    /// The multiplier value to convert from `from_unit` to the Unit.
+    pub fn multiplier_from(&self, from_unit: Unit) -> f64 {
```

Comment:
I don't think that this method makes things more clear. If we want to keep something like this, I would rather use something like `Unit::conversion_factor_seconds()` (without any arguments), otherwise the number of cases grows quadratically with the number of units. We only use `multiplier_from(Second)` anyway.

Reply:
Good point, I think got here after some other changes and it made sense at the time, but looking at it now it's overkill :)

Removed, and just converting the values directly in `Unit.format`.